.github/pull_request_template.md

```diff
@@ -1,14 +1,35 @@
-< Jira issue link, if applicable >
+_Jira issue link, if applicable_
 
-### Description
+### 🎯 Goal
 
-Describe what this PR changes, why we're making the change, how it should be tested...
+_Describe why we are making this change_
 
-### Checklist
+### 🛠 Implementation details
```

Comment:
Maybe just `Changes`. The `Goal` will cover the `why`. The `Changes` section will cover `what` and `how`. E.g.
`Goal`
Improve sign-in form
`Changes`
Added Csrf token to the template to prevent Csrf attack

Reply:
I think "Implementation details" is descriptive enough, I wouldn't add the "why" of the task in that section, only the "what" and "how", as you suggested.